src/shipit_static_analysis/shipit_static_analysis/clang.py

```diff
@@ -184,7 +206,8 @@ def parse_issues(self, clang_output):
                     logger.info('Skipping clang-diagnostic-error: {}'.format(issue))
                 else:
                     issues.append(issue)
-                    logger.info('Found code issue {}'.format(issue))
+                    mode = issue.is_third_party() and '3rd party' or 'Mozilla'
+                    logger.info('Found {} code issue {}'.format(mode, issue))
```

Comment:
Nit: It may be a little weird to see many lines like `Found 123 code issue Mozilla` (we might wonder what "Mozilla" means here). Maybe something like `issue.is_third_party() and '(third-party)' or '(in tree)'` would be clearer? Not sure what the best `mode` is for "not third party" though.